scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -12,10 +12,25 @@ import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
+import scala.meta.internal.semanticdb.{SymbolInformation, TypeRef, TypeSignature}
 import scalafix.internal.v0._
 
 class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig) extends Printer {
   def pprint(tpe: s.Type): Unit = {
+    def equivalent(name: Option[n.Name], sym: String): Boolean = {
+      name
+        .flatMap(env.lookupInfo)
+        .exists { info =>
+          info match {
+            case SymbolInformation(_, _, kind, _, _, TypeSignature(_, TypeRef(_, lbsym, _), TypeRef(_, ubsym, _)), _, _) =>
+              lbsym == ubsym && lbsym == sym
```

Comment:
The idiomatic names for lower and upper bounds are `lo`, `hi` and variations thereof.